plugins/module_utils/redfish_utils.py

```diff
@@ -63,11 +69,23 @@ def get_request(self, uri):
         req_headers = dict(GET_HEADERS)
         username, password, basic_auth = self._auth_params(req_headers)
         try:
-            resp = open_url(uri, method="GET", headers=req_headers,
-                            url_username=username, url_password=password,
-                            force_basic_auth=basic_auth, validate_certs=False,
+            #To support HPE operations
+            if os.path.isfile(home + "/sessionfile.txt"):
+            # if "Hpe" in self.creds.keys():
+                f = open(home + "/sessionfile.txt")
```

Comment:
Never use a raw `open` in Python. Use a `with` construct.

Reply:
Changes have been made. This can be seen in the latest commit of the PR.